src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -231,12 +231,17 @@ def setProcessingVer(self, procVer):
 
         Set the block's processing version.
         """
-        # compatibility statement for old style proc ver (still needed ?)
-        if procVer.count("-") == 1:
-            self.data["processing_era"]["processing_version"] = procVer.split("-v")[1]
+        pver = procVer
```

Comment:
I think I commented on this code in the other PR.
When it gets to this level, processing version has been extensively tested. Maybe the only thing that we would need to keep here is something like:
```
if ver is None:
    procVer = 0
```

in case this information isn't loaded from the database.

Reply:
The same argument here again. It is much safer to check explicitly the provided data type and avoid potential issue with some caller which will not depend on database type definition.